google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITSystemTest.java

```diff
@@ -1190,4 +1191,55 @@ public void floatIncrement() throws ExecutionException, InterruptedException {
     DocumentSnapshot docSnap = docRef.get().get();
     assertEquals(3.3, (Double) docSnap.get("sum"), DOUBLE_EPSILON);
   }
+
+  @Test
+  public void getAllWithObserver() throws Exception {
+    DocumentReference ref = randomColl.document("doc1");
+    ref.set(ALL_SUPPORTED_TYPES_MAP).get();
+
+    final List<DocumentSnapshot> documentSnapshots = new ArrayList<>();
+    final DocumentReference[] documentReferences = {ref};
+    firestore.getAll(
+        FieldMask.of("foo"),
+        new ApiStreamObserver<DocumentSnapshot>() {
+
+          @Override
+          public void onNext(DocumentSnapshot documentSnapshot) {
+            documentSnapshots.add(documentSnapshot);
+          }
+
+          @Override
+          public void onError(Throwable throwable) {}
```

Comment:
I think confirming this isnt being called would be helpful.

`Assert.fail("No error occurred while streaming");`

Reply:
done